src/views/kitchen-sink/page.php

```diff
@@ -242,11 +242,18 @@ class="tribe-events-c-search"
 				class="tribe-common-form-control-text tribe-events-c-search__input-control tribe-events-c-search__input-control--keyword"
 				data-js="tribe-events-events-bar-input-control"
 			>
+				<em
+					class="tribe-events-c-search__input-control-icon"
+					aria-label="Search"
+					title="Search"
+				>
+					<?php $this->template('components/icons/search', [ 'classes' => ['tribe-events-c-search__input-control-icon-svg'] ] ) ?>
```

Comment:
* :no_entry_sign: Inline PHP statement must end with a semicolon
* :no_entry_sign: Expected 1 spaces after opening parenthesis; 0 found
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: Missing space after array opener.
* :no_entry_sign: Missing space after array opener.
* :no_entry_sign: Missing space before array closer.
* :no_entry_sign: Missing space before array closer.

<sub>hash: b271402cbd467bfd0a2154c6e8a2f699</sub>